### PR TITLE
chore: Add billing veneer library

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

File: google-cloud-billing/.repo-metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "secret_manager",
```
Comment (quartzmo) [inline google-cloud-billing/.repo-metadata.json]: Replace Secret Manager naming everywhere in this file.

File: google-cloud-billing/lib/google/cloud/billing.rb [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google-cloud-billing"
+
+module Google
+  module Cloud
+    ##
+    # Allows developers to manage billing for their Google Cloud Platform projects programmatically
```
Comment (quartzmo) [inline google-cloud-billing/lib/google/cloud/billing.rb]: I'd suggest ending this sentence with a period.

File: google-cloud-billing/.repo-metadata.json (hunk 1 above)
Comment (dazuma) [reply google-cloud-billing/.repo-metadata.json]: Whups, missed those. Thanks!

File: google-cloud-billing/lib/google/cloud/billing.rb (hunk 2 above)
Comment (dazuma) [reply google-cloud-billing/lib/google/cloud/billing.rb]: Done

**Pull request merged**